### PR TITLE
Add note shortcode to 'Debugging Kubernetes nodes with crictl'

### PR DESCRIPTION
**Pull request by reylejano**
Fixes [k/website issue 25504](https://github.com/kubernetes/website/issues/25504)
Add note shortcode to 'Debugging Kubernetes nodes with crictl'

### CODE REVIEW

**Comment by reylejano** [comment]
The preview for the edited page:
https://deploy-preview-25507--kubernetes-io-master-staging.netlify.app/docs/tasks/debug-application-cluster/crictl/

**Review by feiskyer — APPROVED** [review]
/lgtm

File: content/en/docs/tasks/debug-application-cluster/crictl.md [hunk 1]
```diff
@@ -360,7 +360,9 @@ for more information.
 
 The exact versions for below mapping table are for docker cli v1.40 and crictl v1.19.0. Please note that the list is not exhaustive. For example, it doesn't include experimental commands of docker cli.
 
+{{< note >}}
 Warn: the output format of CRICTL is similar to Docker CLI, despite some missing columns for some CLI. Make sure to check output for the specific command if your script output parsing.
+{{< /note >}}
```
Comment (sftim) [inline content/en/docs/tasks/debug-application-cluster/crictl.md:365]: I'd write:
```suggestion
{{< note >}}
The output format of CRICTL is similar to Docker CLI, despite some missing columns for some CLI. Make sure to check output for the specific command if your script output parsing.
{{< /note >}}
```

**Review by sftim — COMMENTED** [review]
Thanks @reylejano 
/approve

**Pull request merged**